### PR TITLE
Fix golint failures in pkg/kubelet/lifecycle

### PR DESCRIPTION
**Pull request by yan-dblinf**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Fixes golint failures in pkg/kubelet/lifecycle
Ref: #68026

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by yan-dblinf** [comment]
Signed the CLA

**Comment by yan-dblinf** [comment]
@matthyx Thanks for the review. I updated the missing change on `hack/.golint_failures`, and passed checks. PTAL.

**Comment by matthyx** [comment]
Fine for me, thx!
/lgtm

File: pkg/kubelet/lifecycle/handlers.go [hunk 1]
```diff
@@ -171,6 +174,8 @@ func (a *appArmorAdmitHandler) Admit(attrs *PodAdmitAttributes) PodAdmitResult {
 	}
 }
 
+// NewNoNewPrivsAdmitHandler returns a PodAdmitHandler which is used to evaluates
```
Comment (SergeyKanzhelev) [inline pkg/kubelet/lifecycle/handlers.go:177]:
```suggestion
// NewNoNewPrivsAdmitHandler returns a PodAdmitHandler which is used to evaluate
```

File: pkg/kubelet/lifecycle/handlers.go [hunk 2]
```diff
@@ -236,6 +241,8 @@ func noNewPrivsRequired(pod *v1.Pod) bool {
 	return false
 }
 
+// NewProcMountAdmitHandler returns a PodAdmitHandler which is used to evaluates
```
Comment (SergeyKanzhelev) [inline pkg/kubelet/lifecycle/handlers.go:244]:
```suggestion
// NewProcMountAdmitHandler returns a PodAdmitHandler which is used to evaluate
```

File: pkg/kubelet/lifecycle/handlers.go [hunk 3]
```diff
@@ -144,6 +145,8 @@ func getHttpRespBody(resp *http.Response) string {
 	return ""
 }
 
+// NewAppArmorAdmitHandler returns a PodAdmitHandler which is used to evaluates
```
Comment (SergeyKanzhelev) [inline pkg/kubelet/lifecycle/handlers.go:148]:
```suggestion
// NewAppArmorAdmitHandler returns a PodAdmitHandler which is used to evaluate
```

**Comment by yan-dblinf** [comment]
@SergeyKanzhelev Thanks for correcting typo

**Review by SergeyKanzhelev — APPROVED** [review]

**Comment by yan-dblinf** [comment]
Hi @dims, could you approve the PR if it looks good to you? It has been submitted for three weeks.

**Comment by dims** [comment]
/approve

(for hack/.golint_failures changes only)

**Comment by yan-dblinf** [comment]
@dims Thank you for the quick reply and approval!

**Pull request merged**